service-tests/requires.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const isRequireStatus = Joi.string().regex(/^(up to date|outdated|insecure)$/);
+
+const t = new ServiceTester({ id: 'requires', title: 'Requires.io' });
+module.exports = t;
+
+
+t.create('requirements (valid, without branch)')
+  .get('/github/celery/celery.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'requirements',
+    value: isRequireStatus
+  }));
+
+t.create('requirements (valid, with branch)')
+  .get('/github/celery/celery/master.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'requirements',
+    value: isRequireStatus
+  }));
+
+t.create('requirements (not found)')
+  .get('/github/PyvesB/EmptyRepo.json')
+  .expectJSON({name: 'requirements', value: 'unknown'});
+
+t.create('requirements (connection error)')
+  .get('/github/celery/celery.json')
+  .networkOff()
+  .expectJSON({name: 'requirements', value: 'inaccessible'});
+
+t.create('requirements (unexpected response)')
+  .get('/github/celery/celery.json')
+  .intercept(nock => nock('https://requires.io/')
+    .get('/api/v1/status/github/celery/celery')
+    .reply(200, "{{{{{invalid json}}")
+  )
+  .expectJSON({name: 'requirements', value: 'unknown'});
```

Comment:
This is the same message as the 404 case. Could you change this one to `'invalid'` and the 404 case to `'not found'`?